languages/CMakeLists.txt

```diff
@@ -9,10 +9,10 @@ file(GLOB_RECURSE SOURCES
 	"${CMAKE_CURRENT_LIST_DIR}/../gui/src/*.ui"
 	"${CMAKE_CURRENT_LIST_DIR}/../cli/src/*.cpp")
 qt5_create_translation(QM_FILES ${SOURCES} ${TS_FILES})
-add_custom_target(General_translations DEPENDS ${QM_FILES})
+add_custom_target(General_translations ALL DEPENDS ${QM_FILES})
 
 if(APPLE AND UNIX)
 	install(FILES ${QM_FILES} DESTINATION Grabber.app/Contents/Resources/languages)
 else()
-	install(FILES ${QM_FILES} DESTINATION ${CMAKE_INSTALL_PREFIX}/languages)
+	install(FILES ${QM_FILES} DESTINATION ${CMAKE_INSTALL_PREFIX}/share/Grabber/languages)
```

Comment:
Same remark for `${CMAKE_INSTALL_PREFIX}`.
I actually didn't know it was optional ^^'